src/components/MapView/Analyser/index.tsx

```diff
@@ -70,14 +75,42 @@ async function submitAnaysisRequest(
 }
 
 function Analyser({ classes }: AnalyserProps) {
-  const [open, setOpen] = useState(false);
-  const [selectedBaselineLayer, setSelectedBaselineLayer] = useState(
-    baselineLayers[0],
+  const [open, setOpen] = useState(true);
+  const [analyserOption, setAnalyserOption] = useState('new');
+  const [hazardLayerId, setHazardLayerId] = useState(
```

Comment:
Wouldn't it be better to store the actual layer object instead of the id? It'll make things simpler, otherwise we'll be referencing `LayerDefinitions` every time we want to access the layer. We can store the id in redux once we complete an analysis request.

Reply:
Using the `Id` because every time when we change the the options of layer of which we have run the analysis, we have to fetch the object of that option and assign to the state it becomes a lot of call for the function. So, it just assign the Id of the layer to the state and fetch the object for the layer when we make the `runAnalysis` call. This make state store the less amount of data and make it more serializable.

Reply:
sounds good to me :+1: